packages/curve-ui-kit/src/shared/ui/LargeTokenInput.tsx

```diff
@@ -102,7 +102,7 @@ export interface LargeTokenInputRef {
  * @property {boolean} [showBalance=true] - Whether to display the balance component.
  *                                        When true, shows the token balance and notional value.
  *                                        When false, hides the balance display.
- * @property {boolean} [showSlider=true] - Whether to display the percentage slider.
+ * @property {boolean} [showSlider=false] - Whether to display the percentage slider.
```

Comment:
should we also remove `showSlider: false` from other places in the code? It should be a simple search+replace.